src/marqo/tensor_search/tensor_search.py

```diff
@@ -1003,8 +1000,12 @@ def _vector_text_search(
         # empty body means that there are no vector fields associated with the index.
         # This probably means the index is emtpy
         return {"hits": []}
+
     response = HttpRequests(config).get(path=F"{index_name}/_msearch", body=utils.dicts_to_jsonl(body))
 
+    if verbose:
+        print(f'Opensearch reported {response["took"]}ms search latency')
```

Comment:
Can this actually be logged as an INFO message?

Reply:
Sure, can update